fastlane/lib/fastlane/plugins/plugin_generator.rb

```diff
@@ -82,5 +84,12 @@ def replace_path_variables(template_path, plugin_info)
 
       path
     end
+
+    # This method takes care of any cleaning up we have to do
+    # after copying over all the files
+    def after_all(dest_path)
+      # Prefix the rubocop.yml file with a dot, since it's a rubocop requirement
+      FileUtils.mv(File.join(dest_path, "rubocop.yml"), File.join(dest_path, ".rubocop.yml"))
+    end
```

Comment:
Unfortunately, those files are ignored when building the gem, though, copying them over only works while developing at the moment